index.js

```diff
@@ -227,14 +227,18 @@ function requestAsEventEmitter(opts) {
 					total: uploadBodySize
 				});
 
-				if (req.connection) {
-					req.connection.once('connect', () => {
+				const socket = req.connection;
+				if (socket) {
+					// `._connecting` was the old property which was made public in node v6.1.0
+					const isConnecting = socket._connecting || socket.connecting;
```

Comment:
You should check for `undefined` here explicitly.

Reply:
@floatdrop fixed